.github/workflows/ci.yml

```diff
@@ -8,8 +8,8 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.7", "3.8"]
-        django-version: ["3.1.4"]
+        python-version: ["3.8", "3.9"]
```

Comment:
Do we want to add python 3.10 and 3.11 this as well?

Reply:
Replaced with 3.8 and 3.12, it doesn't make much sense to check versions in-between.